<?xml version="1.0" encoding="utf-8"?>
<!DOCTYPE TS><TS version="2.0" language="bg_BG" sourcelanguage="">
<context>
    <name>Plugin</name>
    <message>
        <location filename="dummy.py" line="24"/>
        <source>Dummy plugin</source>
        <translation>Фалшив плъгин</translation>
    </message>
    <message>
        <location filename="dummy.py" line="32"/>
        <source>Dummy</source>
        <translation>Фалшив</translation>
    </message>
    <message>
        <location filename="editor.py" line="138"/>
        <source>Text editor plugin</source>
        <translation>Плъгин за редактиране на текст</translation>
    </message>
    <message>
        <location filename="editor.py" line="164"/>
        <source>Editor</source>
        <translation>Редактор</translation>
    </message>
    <message>
        <location filename="image.py" line="116"/>
        <source>Image viewer plugin</source>
        <translation>Плъгин за преглед на изображения</translation>
    </message>
    <message>
        <location filename="image.py" line="135"/>
        <source>Image</source>
        <translation>Изображение</translation>
    </message>
    <message>
        <location filename="multimedia.py" line="59"/>
        <source>Multimedia</source>
        <translation>Мултимедиа</translation>
    </message>
    <message>
        <location filename="package.py" line="190"/>
        <source>Package manager plugin</source>
        <translation>Плъгин за управление на пакетите </translation>
    </message>
    <message>
        <location filename="package.py" line="203"/>
        <source>Package</source>
        <translation>Пакет</translation>
    </message>
    <message>
        <location filename="quit.py" line="14"/>
        <source>Quit plugin</source>
        <translation>Плъгин за приключване</translation>
    </message>
    <message>
        <location filename="screenshot.py" line="61"/>
        <source>Screenshot taking plugin</source>
        <translation>Плъгин за правене на снимка на екрана</translation>
    </message>
    <message>
        <location filename="screenshot.py" line="74"/>
        <source>Screenshot</source>
        <translation>Снимка на екрана</translation>
    </message>
    <message>
        <location filename="storage.py" line="296"/>
        <source>Storage management plugin</source>
        <translation>Плъгин за управление на хранилищата</translation>
    </message>
    <message>
        <location filename="storage.py" line="318"/>
        <source>Storage</source>
        <translation>Хранилище</translation>
    </message>
    <message>
        <location filename="terminal.py" line="37"/>
        <source>Embed terminal plugin</source>
        <translation>Плъгин за вграден терминал</translation>
    </message>
    <message>
        <location filename="terminal.py" line="50"/>
        <source>Terminal</source>
        <translation>Терминал</translation>
    </message>
    <message>
        <location filename="www.py" line="361"/>
        <source>WWW</source>
        <translation>УУУ</translation>
    </message>
    <message>
        <location filename="archive.py" line="49"/>
        <source>Archive manager plugin</source>
        <translation>Плъгин за управление на архиви</translation>
    </message>
    <message>
        <location filename="archive.py" line="69"/>
        <source>Archive</source>
        <translation>Архив</translation>
    </message>
    <message>
        <location filename="network.py" line="24"/>
        <source>Network manager plugin</source>
        <translation>Плъгин за управление на мрежата</translation>
    </message>
    <message>
        <location filename="network.py" line="32"/>
        <source>Network</source>
        <translation>Мрежа</translation>
    </message>
    <message>
        <location filename="settings.py" line="32"/>
        <source>Settings</source>
        <translation>Настройки</translation>
    </message>
    <message>
        <location filename="multimedia.py" line="36"/>
        <source>Multimedia plugin</source>
        <translation>Мултимедиа плъгин</translation>
    </message>
    <message>
        <location filename="settings.py" line="24"/>
        <source>Settings manager plugin</source>
        <translation>Плъгин за управление на настройките</translation>
    </message>
    <message>
        <location filename="www.py" line="348"/>
        <source>World Wide Web browser plugin</source>
        <translation>Плъгин за разглеждане на World Wide Web</translation>
    </message>
    <message>
        <location filename="mixer.py" line="62"/>
        <source>Mixer</source>
        <translation>Миксер</translation>
    </message>
    <message>
        <location filename="mixer.py" line="49"/>
        <source>Sound mixer manager plugin</source>
        <translation>Плъгин за управление на звуковият миксер</translation>
    </message>
    <message>
        <location filename="calendar.py" line="28"/>
        <source>Calendar plugin</source>
        <translation>Календар плъгин</translation>
    </message>
    <message>
        <location filename="calendar.py" line="41"/>
        <source>Calendar</source>
        <translation>Календар</translation>
    </message>
</context>
<context>
    <name>Widget</name>
    <message>
        <location filename="editor.py" line="19"/>
        <source>CTRL+O</source>
        <translation></translation>
    </message>
    <message>
        <location filename="www.py" line="119"/>
        <source>CTRL+S</source>
        <translation></translation>
    </message>
    <message>
        <location filename="www.py" line="115"/>
        <source>CTRL+R</source>
        <translation></translation>
    </message>
    <message>
        <location filename="www.py" line="117"/>
        <source>CTRL+F</source>
        <translation></translation>
    </message>
    <message>
        <location filename="www.py" line="113"/>
        <source>CTRL+N</source>
        <translation></translation>
    </message>
    <message>
        <location filename="storage.py" line="215"/>
        <source>Open</source>
        <translation>Отвори</translation>
    </message>
    <message>
        <location filename="editor.py" line="95"/>
        <source>Save as</source>
        <translation>Запази като</translation>
    </message>
    <message>
        <location filename="www.py" line="288"/>
        <source>Find</source>
        <translation>Намери</translation>
    </message>
    <message>
        <location filename="www.py" line="315"/>
        <source>Critical</source>
        <translation>Критично</translation>
    </message>
    <message>
        <location filename="screenshot.py" line="25"/>
        <source>Save</source>
        <translation>Запази</translation>
    </message>
    <message>
        <location filename="storage.py" line="214"/>
        <source>Execute</source>
        <translation>Изпулни</translation>
    </message>
    <message>
        <location filename="storage.py" line="216"/>
        <source>Open with</source>
        <translation>Отвори с</translation>
    </message>
    <message>
        <location filename="storage.py" line="218"/>
        <source>Cut</source>
        <translation>Изрежи</translation>
    </message>
    <message>
        <location filename="storage.py" line="219"/>
        <source>Copy</source>
        <translation>Копирай</translation>
    </message>
    <message>
        <location filename="storage.py" line="220"/>
        <source>Paste</source>
        <translation>Постави</translation>
    </message>
    <message>
        <location filename="storage.py" line="222"/>
        <source>Rename</source>
        <translation>Преименувай</translation>
    </message>
    <message>
        <location filename="storage.py" line="223"/>
        <source>Delete</source>
        <translation>Изтрии</translation>
    </message>
    <message>
        <location filename="storage.py" line="224"/>
        <source>Properties</source>
        <translation>Свойства</translation>
    </message>
    <message>
        <location filename="storage.py" line="226"/>
        <source>New file</source>
        <translation>Нов файл</translation>
    </message>
    <message>
        <location filename="storage.py" line="227"/>
        <source>New directory</source>
        <translation>Нова директория</translation>
    </message>
    <message>
        <location filename="www.py" line="110"/>
        <source>CTRL+B</source>
        <translation></translation>
    </message>
    <message>
        <location filename="www.py" line="183"/>
        <source>Untitled</source>
        <translation>Неименуван</translation>
    </message>
    <message>
        <location filename="www.py" line="251"/>
        <source>the remote server refused the connection (the server is not accepting requests)</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="www.py" line="252"/>
        <source>the remote server closed the connection prematurely, before the entire reply was received and processed</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="www.py" line="253"/>
        <source>the remote host name was not found (invalid hostname)</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="www.py" line="254"/>
        <source>the connection to the remote server timed out</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="www.py" line="255"/>
        <source>the operation was canceled via calls to abort() or close() before it was finished.</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="www.py" line="256"/>
        <source>the SSL/TLS handshake failed and the encrypted channel could not be established. The sslErrors() signal should have been emitted.</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="www.py" line="257"/>
        <source>the connection was broken due to disconnection from the network, however the system has initiated roaming to another acess point. The request should be resubmitted and will be processed as soon as the connection is re-established.</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="www.py" line="258"/>
        <source>the connection to the proxy server was refused (the proxy server is not accepting requests)</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="www.py" line="259"/>
        <source>the proxy server closed the connection prematurely, before the entire reply was received and processed</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="www.py" line="260"/>
        <source>the proxy host name was not found (invalid proxy hostname)</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="www.py" line="261"/>
        <source>the connection to the proxy timed out or the proxy did not reply in time to the request sent</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="www.py" line="262"/>
        <source>the proxy requires authentication in order to honour the request but did not accept any credentials offered (if any)</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="www.py" line="263"/>
        <source>the access to the remote content was denied (similar to HTTP error 401)</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="www.py" line="264"/>
        <source>the operation requested on the remote content is not permitted</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="www.py" line="265"/>
        <source>the remote content was not found at the server (similar to HTTP error 404)</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="www.py" line="266"/>
        <source>the remote server requires authentication to serve the content but the credentials provided were not accepted (if any)</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="www.py" line="267"/>
        <source>the request needed to be sent again, but this failed for example because the upload data could not be read a second time.</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="www.py" line="268"/>
        <source>the Network Access API cannot honor the request because the protocol is not known</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="www.py" line="269"/>
        <source>the requested operation is invalid for this protocol</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="www.py" line="270"/>
        <source>an unknown network-related error was detected</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="www.py" line="271"/>
        <source>an unknown proxy-related error was detected</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="www.py" line="272"/>
        <source>an unknown error related to the remote content was detected</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="www.py" line="273"/>
        <source>a breakdown in protocol was detected (parsing error, invalid or unexpected responses, etc.)</source>
        <translation type="unfinished"></translation>
    </message>
    <message>
        <location filename="www.py" line="276"/>
        <source>unknown error</source>
        <translation>неизвестна грешка</translation>
    </message>
    <message>
        <location filename="www.py" line="284"/>
        <source>SSL errors ignored: %s, %s</source>
        <translation>SSL грешка игнорирана: %s, %s</translation>
    </message>
    <message>
        <location filename="www.py" line="294"/>
        <source>Search</source>
        <translation>Търси</translation>
    </message>
    <message>
        <location filename="www.py" line="318"/>
        <source>Info</source>
        <translation>Инфо</translation>
    </message>
    <message>
        <location filename="www.py" line="334"/>
        <source>Back</source>
        <translation>Назад</translation>
    </message>
    <message>
        <location filename="www.py" line="335"/>
        <source>Next</source>
        <translation>Следващ</translation>
    </message>
    <message>
        <location filename="www.py" line="336"/>
        <source>Reload</source>
        <translation>Презареди</translation>
    </message>
    <message>
        <location filename="editor.py" line="95"/>
        <source>Text (*.txt);;All (*)</source>
        <translation>Текст (*.txt);;Всички (*)</translation>
    </message>
    <message>
        <location filename="screenshot.py" line="25"/>
        <source>Image (*.png *.jpg *.jpeg *.svg);;All (*)</source>
        <translation>Изобрайжение (*.png *.jpg *.jpeg *.svg);;Всички (*)</translation>
    </message>
    <message>
        <location filename="www.py" line="315"/>
        <source>Download of &lt;b&gt;%s&lt;/b&gt; failed.</source>
        <translation>Свалянето на &lt;b&gt;%s&lt;/b&gt; се провали.</translation>
    </message>
    <message>
        <location filename="www.py" line="318"/>
        <source>Download of &lt;b&gt;%s&lt;/b&gt; complete.</source>
        <translation>Свалянето на &lt;b&gt;%s&lt;/b&gt; завърши.</translation>
    </message>
    <message>
        <location filename="storage.py" line="71"/>
        <source>File/directory exists</source>
        <translation>Файлът/директорията съществува</translation>
    </message>
    <message>
        <location filename="storage.py" line="71"/>
        <source>File/directory exists, new name:</source>
        <translation>Файлът/директорията съществува, ново име:</translation>
    </message>
    <message>
        <location filename="storage.py" line="201"/>
        <source>Name:</source>
        <translation>Име:</translation>
    </message>
    <message>
        <location filename="archive.py" line="35"/>
        <source>Unsupported format</source>
        <translation>Неподдържан формат</translation>
    </message>
    <message>
        <location filename="storage.py" line="85"/>
        <source>Icons view</source>
        <translation>Преглед като икони</translation>
    </message>
    <message>
        <location filename="storage.py" line="29"/>
        <source>List view</source>
        <translation>Преглед като лист</translation>
    </message>
</context>
</TS>
